<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>780</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>780</width>
    <height>600</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::PreventContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>pxlpeep</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>loupe.ico</normaloff>loupe.ico</iconset>
  </property>
  <property name="whatsThis">
   <string notr="true"/>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>761</width>
     <height>362</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="btnImage0">
      <property name="text">
       <string>Open image...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnImage1">
      <property name="text">
       <string>Open image...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnImage2">
      <property name="text">
       <string>Open image...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnImage3">
      <property name="text">
       <string>Open image...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnImage4">
      <property name="text">
       <string>Open image...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnImage5">
      <property name="text">
       <string>Open image...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnImage6">
      <property name="text">
       <string>Open image...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnImage7">
      <property name="text">
       <string>Open image...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnImage8">
      <property name="text">
       <string>Open image...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnImage9">
      <property name="text">
       <string>Open image...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnExit">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>380</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkSyncWindows">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>301</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Synchronize windows</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCloseAll">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>380</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Close all</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFillScreen">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>380</y>
     <width>250</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Fill screen with open windows</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
